version-control/command-line/tutorial.md

```diff
@@ -62,6 +62,33 @@ $ git log
 
 > Don't forget, you can hit `q` to quit out of a log if it's too long.
 
+By default, with no arguments, git log shows the most recent commits first.
+
+If we want to limit the number of log entries displayed we can use the a number as an option, such as -3 to show only the last three entries. 
+
+```bash
+$ git log -3
+```
+
+A very useful option is the --patch or -p. This command shows the difference of each commit. This can be very helpful to see what has changed on the last commited changes.
+
+```bash
+$ git log -p
+```
+
+If we want to see a few stats for each commit we can use the --stat command.
+
+```bash
+$ git log --stat
+```
+
+Even can do graphs with the --graph command. It will display an ASCII graph of the branch and merge history beside the log output. 
```

Comment:
The first few words here I don't think make sense, maybe a word at the beginning is missing.